sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestSuite.java

```diff
@@ -79,21 +64,12 @@
    * */
   public TestSuite() {
     // Empty constructor is required by xstream
+    this.testSuites = new ArrayList<>();
+    this.testCases = new ArrayList<>();
   }
 
-  /**
-   * Instantiates a new test suite.
-   *
-   * @param name       the name
-   * @param file       the file
-   * @param tests      the tests
-   * @param assertions the assertions
-   * @param time       the time
-   * @param testSuites the test suites
-   * @param testCases  the test cases
-   */
   public TestSuite(final String name, final String file, final String tests, final String assertions,
-                   final double time, final List<TestSuite> testSuites, final List<TestCase> testCases) {
+    final double time, final List<TestSuite> testSuites, final List<TestCase> testCases) {
     super();
```

Comment:
Before my changes, for no good reason, now it is to init the arrays. Anyway, this constructor will likely go.